client/blocks/upload-image/constants.js

```diff
@@ -1 +1,25 @@
-export const ALLOWED_FILE_EXTENSIONS = [ 'jpeg', 'gif', 'png', 'bmp', 'tiff', 'ico' ];
+/**
+ * External dependencies
+ */
+import i18n from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { ValidationErrors } from 'lib/media/constants';
+
+export const ALLOWED_FILE_EXTENSIONS = [ 'jpg', 'jpeg', 'gif', 'png', 'bmp', 'tiff', 'ico' ];
+
+export const ERROR_UNSUPPORTED_FILE = 'ERROR_UNSUPPORTED_FILE';
+export const ERROR_IMAGE_EDITOR_DONE = 'ERROR_IMAGE_EDITOR_DONE';
+export const ERROR_UPLOADING_IMAGE = 'ERROR_UPLOADING_IMAGE';
+
+export const ERROR_STRINGS = {
```

Comment:
Is the `ERROR_STRINGS` going to be used at any place other than _client/blocks/upload-image/index.jsx_? It doesn't look very "constant" to me and maybe it could be completely removed. What if
```js
ERROR_STRINGS[ ERROR_UNSUPPORTED_FILE ]()
```
was just simply replaced with
```js
translate( 'File you are trying...' )
```
? I don't understand why the extra abstraction is needed.